src/test/kotlin/accountTest/GavetaTest.kt

```diff
@@ -0,0 +1,36 @@
+package accountTest
+
+import account.Gaveta
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+
+
+class GavetaTest {
+
+    @Test
+    fun `quando abrir gaveta for sucesso, ira retornar mensagem de valores conferidos`(){
```

Comment:
```suggestion
    fun `quando valor informado pelo cliente e o valor conferido forem iguais, ira retornar mensagem de valores conferidos`(){
```

Nota que pra quem lê o teste, não é claro qual é o conceito de "sucesso". Por isso, é bom ser explícito no que você espera (assim como você fez no teste da linha 21)